pkg/btc-wallet/src/js/components/lib/send.js

```diff
@@ -68,30 +68,39 @@ const Send = ({ stopSending, value, conversion }) => {
   const checkPayee = (e) => {
     setError('');
 
-    let payeeReceived = e.target.value;
-    let isPatp = ob.isValidPatp(payeeReceived);
-    let isAddress = validate(payeeReceived);
-    if (isPatp) {
-      let command = { 'check-payee': payeeReceived };
+    const validPatPCommand = (validPatP) => {
+      let command = { 'check-payee': validPatP };
       api.btcWalletCommand(command);
       setTimeout(() => {
         setCheckingPatp(false);
       }, 5000);
       setCheckingPatp(true);
       setPayeeType('ship');
-      setPayee(payeeReceived);
+      setPayee(validPatP);
+    };
+
+    let payeeReceived = e.target.value;
+    let isPatp = ob.isValidPatp(payeeReceived);
+    let isAddress = validate(payeeReceived);
+    if (isPatp) {
+      validPatPCommand(payeeReceived);
     } else if (isAddress) {
       setPayee(payeeReceived);
       setReady(true);
       setCheckingPatp(false);
       setPayeeType('address');
       setValidPayee(true);
     } else {
-      setPayee(payeeReceived);
-      setReady(false);
-      setCheckingPatp(false);
-      setPayeeType('');
-      setValidPayee(false);
+      const possibleValidPatPMissingSig = '~'.concat(payeeReceived);
```

Comment:
Should just use deSig method (can find it in Landscape) to simplify this flow.

Reply:
I might be misunderstanding what you meant here. `deSig` appears to remove a sig from a string, here we're adding a sig to a string and then testing whether or not that string is a valid patp.

Reply:
Use deSig as follows on the input before testing it:

`~${deSig(input)}`

This will guarantee that input has a leading ~, as deSig has no effect on input without a leading ~.

Reply:
Ah, ok. Updating now.